validate.py

```diff
@@ -127,10 +128,7 @@ def determine_if_mergeable():
       else:
         rejections.append(user)
 
-  if rejections:
-    raise Exception('Rejected by: %s' % (' '.join(rejections)))
-
-  required_approvals = len(users)
+  required_approvals = math.ceil(len(users)/3 * 2)
```

Comment:
The use of `/` threw me for a bit. I thought it might be doing truncating division, so 5 players would only require 2 approvals. In Python 3.x `/` changed to floating point division, so this works as intended as long as we continue using Python 3 exclusively.

Reply:
It's fine for code to assume python 3.  More specifically, we'll stay on 3.5 (https://github.com/jeffkaufman/nomic/blob/master/.travis.yml) until it goes EoL or travis drops support for it.  (We should have v3.5 until ~2020-09 per https://devguide.python.org/#branchstatus)